common/src/reserved.rs

```diff
@@ -65,12 +86,129 @@ impl ReservedState {
             .collect())
```

Comment:
When it comes to `get_validator_set`, if any validator lose their voting power and becomes 0, it has data discrepancy between `consensus_leader_order` and `get_validator_set`. Since the member does only have `consensus_leader_order` not `validator_leader_order`, the method between `get_validator_set` and `get_governance_set` is different from my sense.

Reply:
it seems like the previous modified logic is not required so I reverted here